linux/webcrypto_plugin.cc

```diff
@@ -0,0 +1,70 @@
+#include "include/webcrypto/webcrypto_plugin.h"
```

Comment:
This is just the default template for linux, right?

Reply:
Just like Android, the plugin stuff generated by `flutter create` is kept intact, but is basically a no-op. It's just used to convince Flutter that Linux is supported in `pubspec.yaml`.